scrapbook-gateway-service/routes/album.py

```diff
@@ -17,32 +18,153 @@ def createAlbum():
     @return - http status code
     """
     try:
-        data = request.form
-        print(data, file = sys.stderr)
-        userID = data['userID']
-        #make sure the name of the parameter matches for name of album
-        albumName = data['name']       
-        response = requests.post(f'{IMAGE_SERVICE_URL__DEV}/albums/create', data = data)
+        #data = request.json
+        userID = request.args.get('userid')
+        albumName = request.form.get('name')
+        response = requests.post(f'{IMAGE_SERVICE_URL__DEV}/album?userid={userID}', headers = request.headers, data = request.data)
         response.raise_for_status()
         return response.json(), response.status_code
 
     except requests.exceptions.HTTPError as err:
         return err.response.text, err.response.status_code
 
 
-# @album_user_api.route('/albums/create', methods=["POST"])
-# @session_service.checkUserSession()
-# def getAlbums():
-#     """
-#     Get all the albums of the user
+album_list_api = Blueprint('album_list_api', __name__)
 
-#     @params - A GET request that is sent to fetch all the albums that the user has access to
-#     @return - http status code
-#     """
-#     try:
-#         response = requests.post(f'{IMAGE_SERVICE_URL__DEV}/albums/user', data = request.json)
-#         response.raise_for_status()
-#         return response.json(), response.status_code
+@album_list_api.route('/album', methods=["GET"])
+@session_service.checkUserSession()
+def getAlbums():
+    """
+    Get all the albums of the user
+
+    @params - A GET request that is sent to fetch all the albums that the user has access to
```

Comment:
Params = parameters. In this, you'll write what are the INPUTS. What you've written is the description. Please update that everywhere.

Here the param will be userid. But you also have to document the request data.

Reply:
Okay cool, what does document the request data mean exactly? The schema?